plugin-server/src/main/ingestion-queues/queue.ts

```diff
@@ -25,10 +25,14 @@ export async function startQueues(
     workerMethods: Partial<WorkerMethods> = {}
 ): Promise<Queues> {
     const mergedWorkerMethods = {
-        runAsyncHandlersEventPipeline: (event: IngestionEvent) => {
+        runAsyncHandlersEventPipeline: (event: PostIngestionEvent) => {
             server.lastActivity = new Date().valueOf()
             server.lastActivityType = 'runAsyncHandlersEventPipeline'
-            return piscina.run({ task: 'runAsyncHandlersEventPipeline', args: { event } })
+            const clonableEvent: ClonableIngestionEvent = {
```

Comment:
Note to self: This is not good - we're transforming the same event N times on the way to the event pipeline, with this particular transformation now being unchanged. This will cause bugs.